shell/store/type-map.js

```diff
@@ -222,6 +224,26 @@ export function DSL(store, product, module = 'type-map') {
     },
 
     headers(type, headers) {
+      const extensionCols = store.$plugin.getUIConfig(ExtensionPoint.TABLE_COL, TableColumnLocation.RESOURCE);
+
+      // Try and insert the columns before the Age column, if that is the last column
+      let insertPosition = headers.length;
+
+      if (headers.length > 0) {
+        const lastColumn = headers[headers.length - 1];
+
+        if (lastColumn?.name === 'age') {
```

Comment:
Pedantic request. Can we use `AGE.name`?

Reply:
Done